bin/mailmap_check.py

```diff
@@ -136,7 +136,8 @@ def key(line):
     if problems:
         print(red(filldedent("""
         For instructions on updating the .mailmap file see:
-        https://github.com/sympy/sympy/wiki/Development-workflow#update-mailmap""")))
+        https://github.com/sympy/sympy/wiki/Development-workflow#update-mailmap""",
```

Comment:
It actually needs to be:

https://github.com/sympy/sympy/wiki/Development-workflow#add-your-name-and-email-address-to-the-mailmap-file

I added that URL before the section existed so I didn't know what the correct URL would be.